apps/aecore/src/aec_peers.erl

```diff
@@ -658,6 +667,13 @@ connected_peers(outbound, #state{ conns = Conns }) ->
     [ peer_info(P) || #conn{ peer = P, type = outbound, state = connected }
                       <- maps:values(Conns) ].
 
+-spec random_connected_peers(N :: pos_integer(), all | inbound | outbound, state())
+    -> [peer_info()].
+random_connected_peers(N, Tag, State) ->
+    Keyed = [{rand:uniform(), P} || P <- connected_peers(Tag, State)],
+    Sorted = lists:keysort(1, Keyed),
```

Comment:
Why do you sort them? I would expect that sorting indirectly implies that certain nodes will get more gossipo than others.... not sure if that has any negative effect for the network as a whole.

We might need to experiment a bit with this.

Reply:
This is to shuffle them, they are sorted on the random key.

Reply:
Good!